services/bit/bit-components.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+const { metric } = require('../text-formatters')
+
+const ownerSchema = Joi.object({
+  payload: Joi.object({
+    totalComponents: Joi.number(),
+  }),
```

Comment:
Do you know if it's possible for the API to return a response object that is missing the `payload` object entirely, or the `payload` object to be missing the `totalComponents` key?

If not, then we should make `payload` a required object, and for `totalComponents` you should be able to use the `nonNegativeInteger` helper from `validators.js`

If those scenarios _are_ possible then we'll need to make sure those are handled in the code below

Reply:
it's should always return object with payload and totalComponents, unless the collection is not exists then it will return 404 with only payload object.
([https://api.bit.dev/scope/ramda/ramda2](https://api.bit.dev/scope/ramda/ramda2))

I have fixed the schema and using